src/Goerli-DssSpell.sol

```diff
@@ -16,29 +16,79 @@
 
 pragma solidity 0.6.12;
 // Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
-// pragma experimental ABIEncoderV2;
+pragma experimental ABIEncoderV2;
```

Comment:
This can be commented out again, it can also be removed from the following files:
```
src/Goerli-DssSpellCollateral.sol
src/test/addresses_deployers.sol
src/test/addresses_goerli.sol
src/test/config.sol
src/test/rates.sol
```

Reply:
Is there any issue with just leaving this turned on always..?

Reply:
I think there was some problem verifying the spell in Etherscan if we added the pragma and wasn't being actually used. I don't remember exactly how it was but it took us a lot of time to realize what was the issue. I have no idea if it is still a problem.

Reply:
ok will remove